taar/recommenders/recommendation_manager.py

```diff
@@ -56,6 +56,10 @@ def recommend(self, client_id, limit):
                     logger.info("No recommendations", extra={
                         "client_id": client_id, "recommender": str(r)
                     })
+                else:
+                    logger.info("Recommendations served", extra={
+                        "client_id": client_id, "recommended addons": str(recommendations)
```

Comment:
While using spaces in dictionary keys is perfectly fine for Python, I'm not sure if this would have any upstream consequence for accessing the logs through STMO. I'd suggest to change the property name to `"recommended_addons"`.

Reply:
oh! nice catch.